BingoDisplay.py

```diff
@@ -255,6 +255,7 @@ def getDefaultPath():
         # Linux
         Path.home() /'snap'/'steam'/'common'/'.local'/'share'/'Steam'/'steamapps'/'common'/'Deus Ex'/'System',
         Path.home() /'.steam'/'steam'/'SteamApps'/'common'/'Deus Ex'/'System',
+        Path.home() /'.local'/'share'/'Steam'/'steamapps'/'compatdata'/'6910'/'pfx'/'drive_c'/'users'/'steamuser'/'Documents'/'Deus Ex'/'System',
```

Comment:
we should also add the non-documents path, for people using Han's Launchbox instead of Kentie's Launcher

```suggestion
        Path.home() /'.local'/'share'/'Steam'/'steamapps'/'compatdata'/'6910'/'pfx'/'drive_c'/'users'/'steamuser'/'Documents'/'Deus Ex'/'System',
        Path.home() /'.local'/'share'/'Steam'/'SteamApps'/'common'/'Deus Ex'/'System',
```